### DISCUSSION TITLE
v1.6.0 Release

### DISCUSSION THREAD

**Discussion by roeiedri** [Chooloo/koler#506] [Announcements]
So there's a new release.
Feel free to roast us or praise us.
❤️ .

**Comment by mmtxrc** [comment]
roast: if we're going forward with THIS id happily stick with 1.5.0 for life.
- not only yes/no but all prompts dont leave the screen unless tapped outside. when i select a sim for calling the popup persists on screen.
- group similar is broken
- images are smaller. yikes
- whatsapp accounts show a number twice
- bigger buttons? who favors bigger buttons?

praise(?): you guys can do better. much better than this. i believe in y'all

**Reply by Jobatphysics** [reply]
Please add call recording button as aosp dialer have
